desc/objectives/_coils.py

```diff
@@ -1412,6 +1411,11 @@ def build(self, use_jit=True, verbose=1):
 
         """
         eq = self._eq
+        self._use_vector_potential = True
+        try:
+            self._field.compute_magnetic_vector_potential([0, 0, 0])
+        except NotImplementedError:
```

Comment:
Is this the only error we'd expect? I thought if a spline field doesn't have A it raises a ValueError?

Reply:
good catch, thanks